data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```diff
@@ -2165,14 +2165,6 @@
       { "item": "touristmap", "prob": 1 }
     ]
   },
-  {
```

Comment:
only used once in the entire game. I took the step of removing it instead of modifying it to add the other painkillers in it.